packages/frontend/src/features/sqlRunner/components/LineChartConfiguration.tsx

```diff
@@ -0,0 +1,55 @@
+import { ActionIcon, Divider, Group, Stack, Title } from '@mantine/core';
+import { IconChevronDown, IconChevronRight } from '@tabler/icons-react';
+import { useState } from 'react';
+import MantineIcon from '../../../components/common/MantineIcon';
+import { LineChartFieldConfiguration } from './LineChartFieldConfiguration';
+import { LineChartStyling } from './LineChartStyling';
+
+export const LineChartConfig = () => {
```

Comment:
We can have a shared `Config` component that takes the current config?

Reply:
Yea, for sure

Reply:
I came up with *an* answer here. See what you think.

Reply:
looks great!